commit ac40b84ee138d45379046e21e31d237f6937a014
Author: Hubert Tarasiuk <hubert.tarasiuk@gmail.com>
Date:   Fri Apr 10 11:52:34 2015 +0200

    Fix error in free_vec.
    
    * src/utils.c (free_vec): Increment pointer instead of its value.
    
    Reported-by: Gisle Vanem <gvanem@yahoo.no>

diff --git a/src/utils.c b/src/utils.c
index 617f547..3c8f4f9 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -1284,7 +1284,10 @@ free_vec (char **vec)
     {
       char **p = vec;
       while (*p)
-        xfree (*p++);
+        {
+          xfree (*p);
+          p++;
+        }
       xfree (vec);
     }
 }
